src/WorkItemMigrator/tests/Migration.Wi-Import.Tests/WitClient/WitClientUtilsTests.cs

```diff
@@ -246,24 +250,28 @@ public void When_calling_ensure_date_fields_with_first_revision_Then_dates_are_a
             MockedWitClientWrapper witClientWrapper = new MockedWitClientWrapper();
             WitClientUtils wiUtils = new WitClientUtils(witClientWrapper);
 
+            WorkItem createdWI = wiUtils.CreateWorkItem("User Story");
+
+            DateTime now = DateTime.Now;
+
             WiRevision rev = new WiRevision();
             rev.Fields = new List<WiField>();
+            rev.Time = now;
             rev.Index = 0;
 
-            WorkItem createdWI = wiUtils.CreateWorkItem("User Story");
-            createdWI.Fields[WiFieldReference.ChangedDate] = DateTime.Now;
+            createdWI.Fields[WiFieldReference.ChangedDate] = now;
 
             wiUtils.EnsureDateFields(rev, createdWI);
 
             Assert.That(rev.Fields[0].ReferenceName, Is.EqualTo(WiFieldReference.CreatedDate));
```

Comment:
Wrap with Assert.Multiple(), and check other places in this file too

Reply:
Fixed by https://github.com/solidify/jira-azuredevops-migrator/pull/648/commits/8ae8dbbb8044c73218f6860e06999ca363f3b068